catalog/video/video-security/src/test/java/org/codice/alliance/video/security/claims/videographer/VideographerClaimsHandlerTest.java

```diff
@@ -160,14 +161,96 @@ public boolean matches(Object o) {
         assertThat(claimsCollection, containsInAnyOrder(nameClaim, emailClaim, ipClaim));
 
         claimsParameters = new ClaimsParameters();
+        claimsParameters.setPrincipal(mock(VideographerPrincipal.class));
         claimsCollection = claimsHandler.retrieveClaimValues(requestClaims, claimsParameters);
 
         assertThat(claimsCollection, hasSize(2));
 
+    }
+
+    @SuppressWarnings("unchecked")
+    @Test
+    public void testRetrieveClaimsForNonVideographerPrincipal() throws URISyntaxException {
+        VideographerClaimsHandler claimsHandler = new VideographerClaimsHandler();
+        claimsHandler.setAttributes(Arrays.asList(CLAIM1,
```

Comment:
Instead of trying to test all the different possible claim formats in a single swoop, should we have smaller, more specific tests that validates the claims map is properly built based on different attribute lists? It would make it easier to see what scenarios are actually covered, will make test failures easier to understand, and we would be able to use simple, valid attribute lists here and in `testRetrieveClaimsForVideographerPrincipal`.